thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/anomaly/detection/trigger/utils/DataAvailabilitySchedulingConfiguration.java

```diff
@@ -37,6 +36,8 @@
   private long consumerPollTimeInMilli = 5_000; // consumer wait 5 secs by default for the buffer to be filled
   private List<String> dataSourceWhitelist;
   private List<String> filterClassList;
+  private long schedulerDelayPerRunInSec = 300;
```

Comment:
schedulerDelayPerRunInSec -> schedulerDelay

Reply:
I keep the unit naming convention, naming it to `schedulerDelayInSec`.